src/component/Common/LanguageSwitcher.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import i18next from "i18next";
+import { languages } from "../../i18n";
+import { useTranslation } from "react-i18next";
+import { IconButton, Menu, MenuItem, Tooltip } from "@mui/material";
+import TranslateIcon from "@mui/icons-material/Translate";
+
+const LanguageSwitcher: React.FC = () => {
+  const { t } = useTranslation();
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <>
+      <Tooltip title={t("login.switchLanguage")}>
+        <IconButton onClick={handleClick} sx={{ ml: 1 }}>
+          <TranslateIcon />
```

Comment:
Please use consistent icon set: Fluent UI Icon.
For this case, consider https://icones.js.org/collection/fluent?variant=Regular+24&s=trans&icon=fluent:translate-24-regular

First you need to copy the SVG content and add this icon into `src/component/Icons` (You can refer to other icons as an example)